transactionoutbox-testing/src/main/java/com/gruelbox/transactionoutbox/testing/ProcessedEntryListener.java

```diff
@@ -0,0 +1,62 @@
+package com.gruelbox.transactionoutbox.testing;
+
+import com.gruelbox.transactionoutbox.TransactionOutboxEntry;
+import com.gruelbox.transactionoutbox.TransactionOutboxListener;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.CountDownLatch;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Collects an ordered list of tx outbox entries that have been 'processed' i.e. succeeded or failed
+ * in processing.
+ */
+@Slf4j
```

Comment:
Pretty sure this already existed somewhere???!

Reply:
nope, similar OrderedEntryListener but not quite the same